### PR TITLE
multiple reference link added

### PR DESCRIPTION
**Pull request by btensay**
Logic for setting single case multiple reference link is added.

https://tools.hmcts.net/jira/browse/ECM-198

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java [hunk 1]
```diff
@@ -92,13 +90,25 @@ public void addSingleCaseToMultipleLogic(String userToken, CaseData caseData, St
 
     }
 
+    private String createMultipleReferenceLink(CaseData caseData)
+    {
+        var newMultipleReference = caseData.getMultipleReference() != null ?
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java:95]: caseData.getMultipleReference() already is a String. There is no need for toString() here

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java [hunk 2]
```diff
@@ -92,13 +90,25 @@ public void addSingleCaseToMultipleLogic(String userToken, CaseData caseData, St
...
+                caseData.getMultipleReference().toString() : "";
+        var ethosReference = caseData.getEthosCaseReference() != null ?
+                caseData.getEthosCaseReference().toString() : "";
+        var url =  "<a href=\"../cases/case-details/"+ethosReference+"\">"+
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java:99]: You should be consistent with your formatting. Have a space around the +

**Review by jthmcts — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java (hunk 2 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java]: done

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java [hunk 3]
```diff
@@ -92,13 +90,21 @@ public void addSingleCaseToMultipleLogic(String userToken, CaseData caseData, St
 
     }
 
+    private String createMultipleReferenceLink(CaseData caseData)
+    {
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java:94]: You should be consistent with formatting. This opening brace should be on the previous line
https://google.github.io/styleguide/javaguide.html#s4.1-braces

**Review by gauravchawla73463 — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleServiceTest.java [hunk 4]
```diff
@@ -54,6 +55,7 @@ public void setUp() {
         submitMultipleEvents = MultipleUtil.getSubmitMultipleEvents();
         caseIdCollection = new ArrayList<>(Arrays.asList("21006/2020", "245000/2020", "245001/2020"));
         userToken = "authString";
+        multipleRefLink = "<a href=\"../cases/case-details/246000" + "\">" + "246000" + "</a>";
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleServiceTest.java:58]: `multipleRefLink = "<a href=\"../cases/case-details/246000\">246000</a>";`

**Review by jthmcts — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java (hunk 3 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/AddSingleCaseToMultipleService.java]: Noted

**Pull request merged**